task/task.go

```diff
@@ -17,6 +17,7 @@ package task
 
 import (
 	"fmt"
+	"github.com/confluentinc/confluent-kafka-go/schemaregistry"
```

Comment:
nit: how about separating std and 3s party imports?

Reply:
Fixed